packages/react-components/src/components/VideoGallery/DefaultLayout.tsx

```diff
@@ -78,9 +78,10 @@ export const DefaultLayout = (props: DefaultLayoutProps): JSX.Element => {
    * For some components which do not strictly follow the order of the array, we might
    * re-render the initial tiles -> dispose them -> create new tiles, we need to take care of
    * this case when those components are here
+   * Note: If for some reason the current number of videos rendered is more than the maximum allowed, 4 videos will be rendered
```

Comment:
Did this happen in testing? I'm concerned we can get to this point and have `maxRemoteVideoStreams - activeVideoStreams` greater than `maxRemoteVideoStreams` to require the Math.max.
Also, why is the limit 4 here? Isn't `maxRemoteVideoStreams` a flexible number?